jwst/cube_build/cube_build_step.py

```diff
@@ -89,15 +89,13 @@ def process(self, input):
             self.grating = self.grating.lower()
         if not self.coord_system.islower():
             self.coord_system = self.coord_system.lower()
-        if not self.output_type.islower():
-            self.output_type = self.output_type.lower()
+        #if not self.output_type.islower():
```

Comment:
Is it necessary to leave these comments lines in here or can they be removed?

Reply:
removed commented code